packages/edit-site/src/components/global-styles/font-library-modal/style.scss

```diff
@@ -48,6 +48,10 @@ $footer-height: 70px;
 		// Push down so that the progress bar is centered vertically.
 		// It should be 120px (72px modal header height + 48px tab height)
 		padding-top: $header-height + $grid-unit-15 + $grid-unit-60;
+
+		& > .font-library-modal__progressbar {
+			width: 160px;
```

Comment:
What I don't like about this approach is that we have to declare it everywhere. Theoretically, if we have 20 places that use a progress bar, we need this style up to 20 times in 20 different locations. This doesn't seem sustainable and feels repetitive - can it be avoided?

Reply:
Yes, I can understand the concern here. Maybe we could keep the default `max-wdith` of `160px` and allow the user to override with a CSS class? @mirka what do you think?